Server/htdocs/AppController/commands_RSM/api/v2/items/getTypes.php

```diff
@@ -0,0 +1,68 @@
+<?php
+//***************************************************************************************
+// Description:
+//    Get one, multiple or all item types and its associated propertyIDS + NAME
+// REQUEST BODY (JSON OBJECT):
+//  EXAMPLE 1:
+//    -Use the endpoint without any param or body to obtain all of them
+//
+//  EXAMPLE 2
+//    - Add a query param with the specific itemtypeIDs
+//            ?ID=6,7,8
```

Comment:
Other PHPs get the parameters from the body and not from the URL. I think all PHPs should work in the same way, so there should be a JSON structure in the body